pkg/agent/cache/manager.go

```diff
@@ -76,6 +78,8 @@ type manager struct {
 	once             sync.Once
 	doneCh           chan struct{}
 	pipelineCnt      int32
+	aliasSPIFFEID    string
+	magicSPIFFEID    string
```

Comment:
nit: `serverSPIFFEID`

Reply:
These should probably be URL type instead of string

Reply:
`serverSPIFFEID` already exists here and it's a string, that's why I'm also using strings here. I can change it if you feel strongly about it.

Reply:
it's ok, we can fix it later. we'll probably need to refactor all this anyways once the workload api updates land.